app/models/concerns/enum_fuel_typeable.rb

```diff
@@ -0,0 +1,16 @@
+module EnumFuelTypeable
```

Comment:
worth adding to advice page as it has the same enums in same order?

Reply:
Will do 👍